habitat/sims/habitat_simulator/habitat_simulator.py

```diff
@@ -257,41 +263,69 @@ def create_sim_config(
         )
 
         sensor_specifications = []
+        VisualSensorTypeSet = {
```

Comment:
We can programatically get the values from a enum by iterating through them with `.value()` 

Also wouldn't this be better defined by some inheritance structure?

Reply:
Are you saying that instead of checking if it's in a set we should get the value it and compare it with the enum numbers? This is just a way to check which type of SensorSpec to initialize based on the values in the config.